### DISCUSSION TITLE
Issue in compilation(?)

### DISCUSSION THREAD

**Discussion by MohammadHadiAlizadeh** [OpenModelica/OpenModelica#13294] [Q&A]
I am encountering an issue when running a simple pipe-in-series model (`Main`) in OpenModelica. The model neither generates any errors or warnings nor proceeds with the simulation. Interestingly, when I activate the `dpout `variable (a completely dummy parameter not used anywhere in the `Dry_Pipe_General` submodel), the problem resolves.

I tested the same model in Dymola, and it works perfectly without any errors.

**The model file is attached for reference.** Any insights or suggestions on how to resolve this would be greatly appreciated.

I'm using 1.24.2 Official release.
[Pipe_in_series.zip](https://github.com/user-attachments/files/17933966/Pipe_in_series.zip)

**Comment by perost** [comment]
It seems to be failing in the backend for some reason. I don't know if it's an issue with the compiler or the model, but the compiler shouldn't just fail silently regardless so I created an issue (#13295) for it.
